lib/salve/set.js

```diff
@@ -78,7 +78,7 @@ Set.prototype.union = function (s) {
  *
  * @param {Function} f A function that selects values. It is called
  * with each value. If the value happens to be an <code>Array</code>
- * then the function is <emph>applied</emph> to this array. A return
+ * then the function is <em>applied</em> to this array. A return
```

Comment:
I would suggest updating the documentation markup here to make use of Markdown.

```
If the value happens to be an ``Array``, then the function is *applied* to this array.
```

The documentation was originally not using the Markdown plugin so I had to using HTML tags to markup code, italics, bold, etc. Then I added the Markdown plugin so you can still find some documentation that uses the old HTML tags but we should privilege Markdown when we make changes.